code/modules/mob/living/simple_animal/hostile/cat_butcher.dm

```diff
@@ -80,7 +80,18 @@
 				var/obj/item/organ/tail/cat/newtail = new
 				newtail.Insert(L, drop_if_replaced = FALSE)
 				return
-		else if(!L.has_trauma_type(/datum/brain_trauma/severe/pacifism) && L.getorgan(/obj/item/organ/ears/cat) && L.getorgan(/obj/item/organ/tail/cat)) //still does damage. This also lacks a Stat check- felinids beware.
+		else if(!L.getorgan(/obj/item/organ/tongue/cat) && L.stat)
+			if(L.getorgan(/obj/item/organ/tongue))
+				var/obj/item/organ/tongue/tongue = L.getorgan(/obj/item/organ/tongue)
+				visible_message("[src] slices off [L]'s tongue!", "<span class='notice'>You slice [L]'s tongue off.</span>")
+				tongue.Remove(L)
+				tongue.forceMove(get_turf(L))
+			else
+				visible_message("[src] implants a cat tongue into [L]!", "<span class='notice'>You implant a cat tongue into [L].</span>")
```

Comment:
maby change "cat tongue" to the organ name to be more generic incase we want more tongue types in the future

Reply:
yeah but, this message only ever comes from the cat butcher, and all he ever does is convert you to a cat